samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml

```diff
@@ -10,53 +10,17 @@ spec:
       securityContext:
         runAsUser: 0
         capabilities:
-          add: ["CHOWN"]
+          add: 
+            - CHOWN
       command:
         - /bin/bash
       args:
         - -c
         - >
-          chown -R "1000:1000" "/workspace/source" &&
-          chown -R "1000:1000" "/tekton/home"
-      resources:
-        limits:
-          cpu: 500m
-          memory: 1Gi
-        requests:
-          cpu: 250m
-          memory: 65Mi
-    - name: step-detect
-      image: docker.io/paketobuildpacks/builder:full
-      securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
-      command:
-        - /cnb/lifecycle/detector
-      args:
-        - -app=/workspace/source/$(build.source.contextDir)
-        - -group=/layers/group.toml
-        - -plan=/layers/plan.toml
-      resources:
-        limits:
-          cpu: 500m
-          memory: 1Gi
-        requests:
-          cpu: 250m
-          memory: 65Mi
-      volumeMounts:
-        - name: layers-dir
-          mountPath: /layers
-    - name: step-restore
-      image: docker.io/paketobuildpacks/builder:full
-      securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
-      command:
-        - /cnb/lifecycle/restorer
-      args:
-        - -layers=/layers
-        - -cache-dir=/cache
-        - -group=/layers/group.toml
+          chown -R "1000:1000" /workspace/source &&
+          chown -R "1000:1000" /tekton/home &&
+          chown -R "1000:1000" /cache &&
```

Comment:
is this one and the next line needed?

Reply:
I think I tried it without it but saw a permission denied problems. The lines can also be found in the Tekton catalog entry, so I simply added them (see https://github.com/tektoncd/catalog/blob/master/task/buildpacks/0.2/buildpacks.yaml#L76-L77). Looking at the underlying Kubernetes, there is this issue: [Volumes are created in container with root ownership and strict permissions #2630 ](https://github.com/kubernetes/kubernetes/issues/2630). Maybe the `fsGroup` setting in the `securityContext` would also work, but not sure on that. I'd stick with how it looks in the catalog unless you want me to go deeper into this.

Reply:
no need for now, this is fine. Im pretty sure we dont need cache, but not for this PR.